src/Infrastructure.Styles.Analyzer/RMSYN1102SeparateExitConditionsAnalyzerTools/ElseStatementSyntaxWrapperStrategy.cs

```diff
@@ -0,0 +1,53 @@
+﻿// This file is part of the re-motion Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// re-motion Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Infrastructure.Styles.Analyzer.RMSYN1102SeparateExitConditionsAnalyzerTools
+{
+  internal class ElseStatementSyntaxWrapperStrategy : IConditionalStatementWrapperStrategy
+  {
+    public void RemoveElseClause (ref SyntaxNode node)
+    {
+      var elseClause = (ElseClauseSyntax)node;
+      var ifStatement = (IfStatementSyntax)elseClause.Statement;
+      ifStatement = ifStatement.WithElse(null);
+      node = elseClause.WithStatement(ifStatement);
+    }
+
+    public void WithCondition (ref SyntaxNode node, SyntaxNode condition)
+    {
+      var elseClause = (ElseClauseSyntax)node;
+      var ifStatement = (IfStatementSyntax)elseClause.Statement;
+      ifStatement = ifStatement.WithCondition((ExpressionSyntax)condition);
+      node = elseClause.WithStatement(ifStatement);
+    }
+
+    public void WithElse (ref SyntaxNode node, ElseClauseSyntax elseClause)
+    {
+      var tempElseClause = (ElseClauseSyntax)node;
```

Comment:
Why call this `tempElseClause` instead of `elseClause` like it's in the other places?